themes/sfx-bold-theme/components/_product-variant.scss

```diff
@@ -0,0 +1,23 @@
+sfx-product-variant {
+  --product-variant-size: 15px;
+  width: var(--product-variant-size);
+  height: var(--product-variant-size);
+  display: inline-block;
+  background-size: cover;
+  margin: 2px;
+  overflow: hidden;
+  cursor: pointer;
+
+  img {
+    width: auto;
+    height: auto;
+    min-width: var(--product-variant-size);
+    min-height: var(--product-variant-size);
+    max-width: 100%;
+    max-height: 100%;
+  }
+
+  &[hidden] {
```

Comment:
Thoughts on rules with modifiers on the element itself above rules with children selectors?